### ISSUE TITLE
Occasionally some unexpected logs appear

### ISSUE THREAD

**Issue by tao12345666333** [apache/apisix-ingress-controller#873]
```
W0217 00:39:04.785645       1 reflector.go:441] pkg/mod/k8s.io/client-go@v0.22.4/tools/cache/reflector.go:167: watch of *v2beta3.ApisixConsumer ended with: an error on the server ("unable to decode an event from the watch stream: unable to decode watch event: no kind \"ApisixConsumer\" is registered for version \"apisix.apache.org/v2beta3\" in scheme \"pkg/runtime/scheme.go:100\"") has prevented the request from succeeding

```

**Comment by xianshun163** [comment]
@tao12345666333 Hello, do you know what caused this log?Not sure if synchronization will be affected

I used the latest ingress controller and still have this log:  

W0228 14:51:12.613401       1 reflector.go:441] pkg/mod/k8s.io/client-go@v0.22.4/tools/cache/reflector.go:167: watch of *v2beta3.ApisixUpstream ended with: an error on the server ("unable to decode an event from the watch stream: unable to decode watch event: no kind \"ApisixUpstream\" is registered for version \"apisix.apache.org/v2beta3\" in scheme \"pkg/runtime/scheme.go:100\"") has prevented the request from succeeding

**Comment by tao12345666333** [comment]
synchronization will not be affected.

**Comment by tao12345666333** [comment]
There will be some retry mechanism here.  But I don't have time to fix it yet, working on other high priority tasks

**Comment by xianshun163** [comment]
If it can't synchronize, there will be problems.
@tao12345666333   Is there any other way to avoid it ?
 I used v1.3 before, but because of the problem of  "context cancel " #851 ,  I  changed to V1.4.  But this problem will also lead to instability in production.

**Comment by tao12345666333** [comment]
@xianshun163  I will add this one to my list.  Fix it after ending other high priority tasks.

**Comment by xianshun163** [comment]
@tao12345666333 ,   my  crd  yaml config is like below , I define ApisixUpstream and ApisixRoute, is that anything error for my config file?     with the config, the apisix  work   all right, but  except  the err log below print always. 
`
2022-03-04T15:15:13+08:00       error   ingress/apisix_upstream.go:130  failed to get service test-env/xss: service "xss" not found
2022-03-04T15:15:13+08:00       warn    ingress/apisix_upstream.go:210  sync ApisixUpstream failed, will retry  {"object": {"Type":1,"Object":"test-env/xss","Tombstone":null}, "error": "service \"xss\" not found"}
2022-03-04T15:15:13+08:00       error   ingress/apisix_upstream.go:130  failed to get service test-env/xss2: service "xss2" not found
2022-03-04T15:15:13+08:00       warn    ingress/apisix_upstream.go:210  sync ApisixUpstream failed, will retry  {"object": {"Type":1,"Object":"test-env/xss2","Tombstone":null}, "error": "service \"xss2\" not found"}
`

`
---
apiVersion: apisix.apache.org/v2beta3
kind: ApisixUpstream
metadata:
  name: xss
spec:
  timeout:
    connect: 3s
    read: 5s
    send: 5s

---
apiVersion: apisix.apache.org/v2beta3
kind: ApisixRoute
metadata:
  name: xss
spec:
  http:
    - name: rule1
      match:
        paths:
          - /testenv/xss/*
      backends:
        - serviceName: xssservice-svc
          servicePort: 8319
      plugins:
        - name: proxy-rewrite
          enable: true
          config:
            regex_uri:
              - "^/testenv/xss/*"
              - "/xs/${1}"
        - name: prometheus
`

**Comment by tao12345666333** [comment]
#893 will solve the "xxx not found" problem

**Comment by AlinsRan** [comment]
This is because ApisixConsumer is not register in scheme. This problem has been fixed in 1.4.1
* https://github.com/apache/apisix-ingress-controller/pull/833

I will close it.
